### ISSUE TITLE
1.3 CAP: role name changes

### ISSUE THREAD

**Issue by prabalsharma** [SUSE/doc-cap#221]
If someone is relying on role_name to manually upgrade their cluster they should know:
1. diego_locket and diego_access role-names have been removed
2. diego_ssh is a new role-name

**Comment by cjschroder** [comment]
Thanks @prabalsharma, the CAP Deployment Guide already has these changes. Does this also change? (from the Helm chart)

MYSQL_DIEGO_LOCKET_PASSWORD

**Comment by btat** [comment]
@cjschroder I don't believe we have the `diego_access => diego_ssh` change yet as that should be a fairly recent change. It looks some older role names, including `diego_locket`, were brought back in a recent update to the `List of SCF Containers` section.

@prabalsharma can you link the card related to the role name changes please so we have a reference.

**Comment by prabalsharma** [comment]
@btat I think it was changed as a part of some bug fix. Cannot find the card for it though. @jandubois might know

**Comment by jandubois** [comment]
@btat It was changed in https://github.com/SUSE/SCF/commit/9c0b68c04ed0fd0e91bd732c0e93fd7b29c63533

The corresponding Trello card is https://trello.com/c/JmwDHOUp/836-2141-pre-deployed-app-wont-come-up-after-upgrade

**Comment by btat** [comment]
Thanks @jandubois and @prabalsharma!

**Comment by btat** [comment]
Also `api` => `api-group`
